app/assets/src/components/views/bulk_download/BulkDownloadTableRenderers.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import DownloadIcon from "~ui/icons/DownloadIcon";
+import LoadingIcon from "~ui/icons/LoadingIcon";
+
+import cs from "./bulk_download_table_renderers.scss";
+
+export default class BulkDownloadTableRenderers extends React.Component {
+  static renderDownload = ({ rowData }) => {
```

Comment:
I guess this will never be an asynchronous table, right?
Just good practice for renderers to have a behavior when rowData is null, but that will probably never happen with this one.

Reply:
I added a
```
if (!rowData) return null
```

Let me know if returning `null` is not correct here.

Reply:
That works.